packages/venia-ui/lib/components/CartPage/GiftCards/giftCards.css

```diff
@@ -0,0 +1,109 @@
+.root {
+    display: grid;
+    gap: 2em;
+}
+
+.apply_button {
+    composes: root_normalPriority from '../../Button/button.css';
+
+    grid-area: apply_button;
+
+    /* match the height of the input form. */
+    height: 2.25rem;
+    /* match the height of the Field label. */
+    margin-top: 28px;
+
+    width: 2rem;
+}
+
+.balance {
+    align-items: start;
+    display: grid;
+    grid-template-columns: max-content 1fr;
+    grid-area: card_balance;
+    font-style: italic;
+}
+
+.card {
+    align-items: start;
+    display: grid;
+    gap: 1rem;
+    grid-template-columns: 1fr max-content max-content;
+    grid-template-areas:
+        'card_entry apply_button toggle_button'
+        'card_balance card_balance card_balance'
+        'check_balance_button check_balance_button check_balance_button';
+}
+
+.card_input_container {
+}
+
+.cards_container {
+    align-items: baseline;
+    display: grid;
```

Comment:
Consider adding a `gap` here so the items don't have to accomplish it with `margin`.

Reply:
Nice, thanks!

Resolved in https://github.com/magento/pwa-studio/pull/2124/commits/9173c3b25090bdec84cfd7c0d0e4b6205beacf95.